diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java
index 48fb5c1..7dd7248 100644
--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java
+++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModule.java
@@ -481,6 +481,11 @@ public class ImagePickerModule extends ReactContextBaseJavaModule
     else
     {
       imageConfig = getResizedImage(reactContext, this.options, imageConfig, initialWidth, initialHeight, requestCode);
+// ---- Fix for: https://github.com/react-native-community/react-native-image-picker/pull/1049
+      if (imageConfig == null) {
+        responseHelper.invokeError(callback, "Video is not supported in image mode");
+      }
+// ----
       if (imageConfig.resized == null)
       {
         removeUselessFiles(requestCode, imageConfig);
